### PR TITLE
Add support for specular layers

### PR DESCRIPTION
**Pull request by dpogue**
This also allows you to turn off preshade/runtime colouring.

### CODE REVIEW

File: korman/exporter/material.py [hunk 1]
```diff
@@ -304,6 +304,18 @@ def export_texture_slot(self, bo, bm, hsgmat, slot, idx, name=None, blend_flags=
...
+        # Check if this layer uses diffuse/runtime lighting
+        if not slot.use_map_color_diffuse:
+            layer.preshade = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
+            layer.runtime = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
+
+        # Check if this layer uses specular lighting
+        if slot.use_map_color_spec:
+            state.shadeFlags |= hsGMatState.kShadeSpecular
+        else:
+            layer.specular = hsColorRGBA(0.0, 0.0, 0.0, 1.0)
```
Comment (Hoikas) [inline korman/exporter/material.py]: Magic number?

Comment (Deledrius) [reply korman/exporter/material.py]: Do we have premade hsColorRGBA values (e.g. "black")?

Comment (dpogue) [reply korman/exporter/material.py]: There are already several spots throughout this file that use "magic colours" (mostly white and black).

In this case, if we're not using specular lighting, the specular colour should be set to black to match PlasmaMax.

Comment (Hoikas) [reply korman/exporter/material.py]: Whoops, I misread this as saying specular is always black

**Pull request merged**